### PR TITLE
Replace `ResizableTriggerIcon` in `ResizeTrigger` to use `@yamada-ui/lucide` icon

### LINKED ISSUE #3536: Replace `ResizableTriggerIcon` with `LucideIcon`
### Description

Replace `ResizableTriggerIcon` with `LucideIcon`

### Problem Statement/Justification

Replace the icons from `@yamada-ui/icon` with those from `@yamada-ui/lucide`.

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by taroj1205**
Closes #3536 

## Description

Replaced `ResizableTriggerIcon` in `ResizeTrigger` to use `@yamada-ui/lucide` icon.

### CODE REVIEW

**Review by kaehehehe — DISMISSED** [review]
👍🏼

File: packages/components/resizable/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "@yamada-ui/core": "workspace:*",
-    "@yamada-ui/icon": "workspace:*",
+    "@yamada-ui/lucide": "workspace:*",
```
Comment (hirotomoyamada) [inline packages/components/resizable/package.json:50]: Please remove line,

Comment (taroj1205) [reply packages/components/resizable/package.json]: Removed.

**Comment by taroj1205** [comment]
Please review again @kaehehehe @hirotomoyamada.

**Comment by hirotomoyamada** [comment]
> Please review again @kaehehehe @hirotomoyamada.

No, please run `pnpm install` and commit `pnpm-lock.yaml`.

**Comment by taroj1205** [comment]
Thanks @hirotomoyamada, I have pushed the new `pnpm-lock.yaml`, please review.

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**